docs-site/docs/docs/faq.md

```diff
@@ -0,0 +1,19 @@
+### Why Polynote?
+
+See the [Why?](https://github.com/polynote/polynote#why) section of our README for more information.
+
+### Where can I find example notebooks? 
+
+We've created several example notebooks to showcase common use cases for Polynote - to see them, navigate to `./docs-site/docs/docs/examples`. 
```

Comment:
Any way to make that into a proper link? A link to https://github.com/polynote/polynote/tree/master/docs-site/docs/docs/examples would be fine

Reply:
Oh, I think the examples are also pre-populated into the Docker distribution (if anyone is actually using that)